spec/features/user_profiles_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+require 'helpers'
+
+feature 'user can view profile' do
+  let(:game) { FactoryGirl.create(:game) }
+  let(:user) { FactoryGirl.create(:user) }
+
+  scenario 'users can view their profile' do
+    sign_in_as(user)
+    make_review_for_game(game)
```

Comment:
That helper lets us be sure that the review is created by the user we're using. And having a review ensures that reviews are showing up on the profile the way they should. We could make that a separate test if you think we should?